custom-fonts.md

```diff
@@ -95,6 +97,15 @@ Then the app can declare a font like in the example below:
 
 The `lib/` is implied, so it should not be included in the asset path.
 
+In this case, since it is a locally defined font, the TextStyle is created
```

Comment:
is this TextStyle instantiation in the package or in the app? I assume both?

Reply:
The app. Changed the text.